eval/src/vespa/eval/eval/fast_sparse_map.h

```diff
@@ -135,6 +135,26 @@ class FastSparseMap
         _map.insert(std::make_pair(Key(hash), value));
     }
 
+    // used to add a mapping, but in the unlikely case
+    // of hash collision it works like lookup instead.
+    template <typename T>
+    size_t lookup_or_add_mapping(ConstArrayRef<T> addr) {
```

Comment:
I think we should just call this add_mapping and remove the old version altogether.